cpp/src/tests/pagerank/pagerank_test.cu

```diff
@@ -115,34 +111,31 @@ class Tests_Pagerank : public ::testing::TestWithParam<Pagerank_Usecase> {
      // Read
      ASSERT_EQ( (mm_to_coo<int,T>(fpin, 1, nnz, &cooRowInd[0], &cooColInd[0], &cooVal[0], NULL)) , 0)<< "could not read matrix data"<< "\n";
      ASSERT_EQ(fclose(fpin),0);
-    
-    // gdf columns
-    col_src = create_gdf_column(cooRowInd);
-    col_dest = create_gdf_column(cooColInd);
 
-    cugraph::edge_list_view(G.get(), col_src.get(), col_dest.get(), nullptr);
-    cugraph::add_transposed_adj_list(G.get());
+     //  Pagerank runs on CSC, so feed COOtoCSR the row/col backwards.
+     CSR_Result_Weighted<int,T>   result;
+     ConvertCOOtoCSR_weighted(&cooColInd[0], &cooRowInd[0], &cooVal[0], nnz, result);
 
-    cudaDeviceSynchronize();
-    if (PERF) {
-      hr_clock.start();
-      for (int i = 0; i < PERF_MULTIPLIER; ++i) {
-       cugraph::pagerank<int,T>(G.get(), d_pagerank);
-       cudaDeviceSynchronize();
+     cugraph::experimental::GraphCSC<int,T> G(result.rowOffsets, result.colIndices, result.edgeWeights, m, nnz);
```

Comment:
Notice here... we don't have to create columns.  The graph_test.cu test demonstrates that we can operate correctly from cudf::column objects.  But if we have other graph data we can operate on the arrays directly without translating into columns and back out.